providers/class-two-factor-fido-u2f-admin.php

```diff
@@ -230,7 +230,7 @@ public static function show_user_profile( $user ) {
 	 * @static
 	 *
 	 * @param int $user_id User ID.
-	 * @return false
+	 * @return void|never
```

Comment:
This is dubious.
`catch` should do better.

Reply:
This file is pending removal via #439 so not too much effort needs to happen here :)